backend/src/main/java/gov/cdc/usds/simplereport/service/UploadService.java

```diff
@@ -76,7 +73,7 @@ public void processPersonCSV(InputStream csvStream) throws IOException {
                     row.get("Role").toUpperCase(),
                     row.get("Email"),
                     row.get("County"),
-                    race,
+                    row.get("Race").toLowerCase(),
```

Comment:
This looks like it will lowercase `UNK` to `unk`, will that be a problem?

Reply:
this is intended. we store race as `unknown` in the DB and Elegante gives up a file with `Unknown`